client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -20,14 +24,29 @@ const MULTI_EMPTY_VALUE = 'unchanged';
 
 const SINGLE_EMPTY_VALUE = 0;
 
+// This ensures that Emotion's styles are inserted before ours so we can override the default react-select styling
+function EmotionCacheProvider({ children }) {
+  const cache = React.useMemo(
+    () => createCache({
+        key: 'react-select',
+        insertionPoint: document.querySelector('title'),
+      }),
+    []
+  );
+
+  return <CacheProvider value={cache}>{children}</CacheProvider>;
+}
+
 class TreeDropdownField extends Component {
```

Comment:
Most of the changes in this class are a result of changes in the API for `react-select`.
See the upgrade guides here: https://react-select.com/upgrade we went from v1 all the way to v5.